lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,4 +4,77 @@
             data-user-create-subcomment="<%= canCreateSubComment %>">
         <a href="#" class="new-post-btn" role="button"><span class="icon fa fa-edit new-post-icon"></span><%= gettext("New Post") %></a>
     </div>
+
+    <div class="team-detail">
+        <h4><%- gettext("Team Details") %></h4>
+        <% if (isMember) { %>
+            <div class="team-member">
+                <span><%- gettext("You are a member of this team.") %></span>
+            </div>
+        <% } %>
+        <div class="team-members">
+            <span class="sr"><%- gettext("Team members profile") %></span>
+            <div class="members-info"></div>
+        </div>
+
+        <div class="team-capacity">
+            <span class="sr"><%- gettext("Team capacity") %></span>
+            <span><%- membershipText %></span>
+        </div>
+
+        <div class="team-country">
+            <span class="sr"><%- gettext("Team location") %></span>
+            <i class="u-field-icon icon fa fa-globe fa-fw" aria-hidden="true"></i>
+            <span>
+                <%=
+                    interpolate_text(
```

Comment:
Why the need to use `interpolate`? Will `<%- gettext(country) %>` not work here?